src/test/java/net/datafaker/FakerTest.java

```diff
@@ -36,6 +36,11 @@ void examplifyMixed() {
         assertThat(faker.examplify("abc123ABC1zzz")).matches("[a-z]{3}[0-9]{3}[A-Z]{3}[0-9][a-z]{3}");
     }
 
+    @Test
+    void test() {
+        assertThat(9).isBetween(1, 65535);
```

Comment:
That's a nice test :-) (I think that slipped in when testing the ports?)

Reply:
yes you're right, good catch, thanks